app/assets/javascripts/solidus_afterpay/afterpay_checkout_button.js

```diff
@@ -0,0 +1,34 @@
+$(document).bind("afterpay.loaded", function () {
+  function initializeExpressCheckout(selector) {
+    AfterPay.initializeForPopup({
+      countryCode: "US",
```

Comment:
Shouldn't we use the global endpoints for Afterpay? What does countryCode entail?

Reply:
It is not related to the APIs: 

```
This prepares the Afterpay JavaScript to initiate the Afterpay screenflow in the appropriate geographical region. This function accepts one required argument: an object with a required countryCode property. This property must contain the two-character ISO 3166-1 country code of the Merchant account.
```